.github/workflows/nightly-normal-session-e2e-tests.yml

```diff
@@ -210,6 +210,26 @@ jobs:
           ALEPH_NODE_BINARY: ../target/release/aleph-node
         run: ./.github/scripts/test_major_sync.sh
 
+  run-force-reorg-test:
+    needs: [build-production-node-and-runtime]
+    name: Run force reorgs test
+    runs-on: self-hosted
```

Comment:
Can you please use this syntax `[self-hosted, Linux, X64, small] to make sure small runer is used

If small is not enough then please let me know what specs this test needs in terms of CPU / RAM, we'll provide appropriate machine (but this will need some time to set it up)

e.g. as in https://github.com/Cardinal-Cryptography/aleph-node/blob/054225cbab6c06c1af436739fe550dad97a13d35/.github/workflows/_check-code-formatting.yml#L10

Reply:
Ok, it passed on medium instance. Thx